app/views/find-a-record/check-confirm.html

```diff
@@ -240,7 +240,7 @@ <h1 class="nhsuk-heading-l">{{ data.patientName }}’s {{data.vaccine}} vaccinat
         ]
       }) }}
 
-      <a class=nhsuk-body-m
+      <p class=nhsuk-body><a class=nhsuk-link
       href="/find-a-record/delete">Delete this record</a>
```

Comment:
These minor changes aren’t strictly needed, but helpful for consistency:

```suggestion
      <p class="nhsuk-body"><a class="nhsuk-link"
      href="/find-a-record/delete">Delete this record</a></p>
```